lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/text.ejs

```diff
@@ -21,16 +21,17 @@ Sub-service: <% if (requestedSubServices.length > 0) { %>
               No roles selected
             <% } %>
 Status: Approved
+<% if (isMultipleSubServicesReq) { %>
+To access this sub-services, sign into your DfE Sign-in account at <%=signInUrl%>. Or, you can select the button below to sign in to your account.
```

Comment:
please update the content to match the prototype 
![image](https://github.com/DFE-Digital/login.dfe.notifications.jobs/assets/70764046/592b5179-d6c3-4130-8ce9-2bdff179117c)

Reply:
done